system/autoware_auto_msgs_adapter/src/include/adapter_planning.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 The Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef ADAPTER_PLANNING_HPP_
+#define ADAPTER_PLANNING_HPP_
+
+#include "adapter_base.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <autoware_auto_planning_msgs/msg/trajectory.hpp>
+#include <autoware_planning_msgs/msg/trajectory.hpp>
+
+#include <string>
+
+namespace autoware_auto_msgs_adapter
+{
+using Trajectory_auto = autoware_auto_planning_msgs::msg::Trajectory;
+using Point_auto = autoware_auto_planning_msgs::msg::TrajectoryPoint;
+using Trajectory = autoware_planning_msgs::msg::Trajectory;
+
+class AdapterPlanning : public autoware_auto_msgs_adapter::AdapterBase<Trajectory, Trajectory_auto>
+{
+public:
+  AdapterPlanning(
+    rclcpp::Node & node, const std::string & topic_name_source,
+    const std::string & topic_name_target, const rclcpp::QoS & qos = rclcpp::QoS{1})
+  : AdapterBase(node, topic_name_source, topic_name_target, qos)
+  {
+    RCLCPP_DEBUG(
+      node.get_logger(), "AdapterPlanning is initialized to convert: %s -> %s",
+      topic_name_source.c_str(), topic_name_target.c_str());
+  }
+
+protected:
+  Trajectory_auto convert(const Trajectory & msg_source) override
+  {
+    Trajectory_auto msg_auto;
+    msg_auto.header = msg_source.header;
+    Point_auto TrajectoryPoint_auto;
```

Comment:
According to the guidelines, use `lower_snake_case` for variables.
```suggestion
    PointAuto trajectory_point_auto;
```